### ISSUE TITLE
App becomes unresponsive when seeking through videos

### ISSUE THREAD

**Issue by bibin-v2020** [jonjomckay/fritter#147]
Recently upgraded to v2.9.0. App became unresponsive while controlling video playback time.

**Comment by bibin-v2020** [comment]
![Screenshot_2021-06-12-13-14-41-30](https://user-images.githubusercontent.com/84482532/121769325-94f0e600-cb80-11eb-91cb-036d54891b07.jpg)

I can hear the audio but app becomes unresponsive. I've to clear the cache and stop it to start working

**Comment by jonjomckay** [comment]
Sorry about this. Which version of Android are you using, and on which device (or device resolution)?

**Comment by bibin-v2020** [comment]
brand: Realme 3 Pro
device: RMX1851
display: RMX1851_11_C.17
Android Version:10
Resolution : 2340 x 1080

**Comment by jonjomckay** [comment]
Annoyingly, I'm not able to reproduce this at all - even on the same video from the screenshot 😞 

There is an update available to the video library Fritter uses though, so I've applied that update, which could fix the issue... Would you be able to test with the APK from [this GitHub action build](https://github.com/jonjomckay/fritter/runs/2859750832)?

**Comment by bibin-v2020** [comment]
> Annoyingly, I'm not able to reproduce this at all - even on the same video from the screenshot 😞
> 
> There is an update available to the video library Fritter uses though, so I've applied that update, which could fix the issue... Would you be able to test with the APK from [this GitHub action build](https://github.com/jonjomckay/fritter/runs/2859750832)?

Sure!! Will update you soon.

**Comment by selurvedu** [comment]
@jonjomckay I saw that https://github.com/jonjomckay/fritter/commit/3f06c150fce7fc6e2446796ee7cfb86d42f9c744 is included in v2.11.0 so I downloaded it and did a clean install since previous install was from F-Droid.

This version does not start at all, being unable to create the database, mentioning a SQL syntax error in `ALTER TABLE subscription DROP COLUMN verified`. I already reported the problem via the report dialogue.

**Comment by bibin-v2020** [comment]
> > Annoyingly, I'm not able to reproduce this at all - even on the same video from the screenshot 😞
> > There is an update available to the video library Fritter uses though, so I've applied that update, which could fix the issue... Would you be able to test with the APK from [this GitHub action build](https://github.com/jonjomckay/fritter/runs/2859750832)?
> 
> Sure!! Will update you soon.

No luck. Issue is still there

**Comment by bibin-v2020** [comment]
I think new version v2.11.1+20210619 somehow fixed the issue.. Though the experience is not that smooth..

**Comment by bibin-v2020** [comment]
The error I got today, when video got stuck

URL: http://i.is.cc/1alCnZGk.jpg

**Comment by jonjomckay** [comment]
From @Alcidauk in #172:

> **Describe the bug**
> After starting video play, if I try to jump at another timestamp, the whole app freeze. I then need to kill it.
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
>     1. Go to a tweet with embedded video
> 
>     2. Start playing video
> 
>     3. Click on the timeline.
> 
> 
> **Screenshots**
> I don't think a screen will help but I can provide ond if needed.
> 
> **Device**
> Moto Z with GSF, android 10 (lineageOs), fritter 2.10.0.
> 
> Nota: a huge thanks for this amazing app. It's really much simplier to use this app instead of twitter website !

Also from #172, from @JJFlash-IT:

> **Describe the bug**
> After starting video play, if I try to jump at another timestamp, the whole app freeze. I then need to kill it.
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
>     1. Go to a tweet with embedded video
> 
>     2. Start playing video
> 
>     3. Click on the timeline.
> 
> 
> **Screenshots**
> I don't think a screen will help but I can provide ond if needed.
> 
> **Device**
> Moto Z with GSF, android 10 (lineageOs), fritter 2.10.0.
> 
> Nota: a huge thanks for this amazing app. It's really much simplier to use this app instead of twitter website !

**Comment by jonjomckay** [comment]
I've just pushed a change to `master` that I'm really hoping will solve this, but I've not been able to reproduce it consistently (only once, and I wasn't at my computer).

For anyone that is able to test, please try downloading the APK from [here](https://github.com/jonjomckay/fritter/runs/2898593475). Fingers crossed 🤞🏽

**Comment by Alcidauk** [comment]
I just installed it, and it fixes both issues: seek and play again !

Everything's fine now, and this complete video is prefect on my lineage moto phone.

Again a juge thanks, for this app and your hard work to fix issues quickly !

**Comment by JJFlash-IT** [comment]
I hope I downloaded the right APK from your link, it was from a zip file named "apk-release.zip". Anyway it definitely improved, now I can restart and seek into short and longer (like 2 mins) videos.
It still has trouble when going to fullscreen and changing orientation from portrait to landscape, anyway the app sent a ton of reports to Sentry, so maybe those will help.
Absolutely a great improvement, thank you so much!

**Comment by jonjomckay** [comment]
Thanks all, that's great to hear! Unless I hear anything else, I'll consider this fixed in the upcoming v2.12.0. I've seen a few reports come into Sentry regarding the fullscreen issues, so I'll work on them in a separate issue.
